geopyspark-backend/vectorpipe/build.sbt

```diff
@@ -6,7 +6,7 @@ libraryDependencies ++= Seq(
   "org.locationtech.geotrellis" %% "geotrellis-s3-testkit" % Version.geotrellis,
   "org.locationtech.geotrellis" %% "geotrellis-spark"      % Version.geotrellis,
   "org.typelevel"               %% "cats"                  % "0.9.0",
-  "com.azavea" %% "vectorpipe" % "0.2.2"
+  "com.azavea" %% "vectorpipe" % "0.3.0"
```

Comment:
@jbouffard woops, I forgot to check this version got bumped.

Reply:
@echeipesh It did. I published vectorpipe 0.3.0 a bit earlier today.